bundle/src/main/java/com/adobe/acs/commons/synthesizedsling/SynthesizedSlingHttpServletRequest.java

```diff
@@ -0,0 +1,200 @@
+package com.adobe.acs.commons.synthesizedsling;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.request.RequestPathInfo;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.wrappers.SlingHttpServletRequestWrapper;
+
+/**
+ * Used to build internal sling requests to do nice things :)
+ *
+ * @see com.adobe.acs.commons.forms.helpers.impl.synthetics.SyntheticSlingHttpServletGetRequest
+ */
+public class SynthesizedSlingHttpServletRequest extends SlingHttpServletRequestWrapper {
+
+    /** GET */
+    public static final String METHOD_GET = "GET";
```

Comment:
Are these METHOD_* vars needed? It doesnt seem like theyre used in the context of Synthesized ?